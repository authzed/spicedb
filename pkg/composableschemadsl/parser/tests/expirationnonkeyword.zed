caveat expiration(someparam int) {
	someparam == 42
}