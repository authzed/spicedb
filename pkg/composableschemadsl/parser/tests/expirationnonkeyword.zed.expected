NodeTypeFile
  end-rune = 5
  input-source = expiration non-keyword test
  start-rune = 0
  child-node =>
    NodeTypeCaveatDefinition
      end-rune = 5
      input-source = expiration non-keyword test
      start-rune = 0
      child-node =>
        NodeTypeError
          end-rune = 5
          error-message = Expected identifier, found token TokenTypeKeyword
          error-source = expiration
          input-source = expiration non-keyword test
          start-rune = 7
    NodeTypeError
      end-rune = 5
      error-message = Unexpected token at root level: TokenTypeKeyword
      error-source = expiration
      input-source = expiration non-keyword test
      start-rune = 7